app/views/scaptimony_policies/_list.html.erb

```diff
@@ -8,23 +8,23 @@
   <% for policy in @policies %>
     <tr>
       <td>
-          <%= link_to policy.name, scaptimony_policy_dashboard_scaptimony_policy_path(:id => policy.id) %>
+        <%= display_link_if_authorized(policy.name.to_s, hash_for_scaptimony_policy_dashboard_scaptimony_policy_path(:id => policy.id)) %>
       </td>
       <td>
-          <% if !policy.scap_content.nil? %>
-            <%= link_to_if_authorized policy.scap_content.title,
-                  hash_for_edit_scaptimony_scap_content_path(:id => policy.scap_content) %>
-          <% end %>
+        <% if !policy.scap_content.nil? %>
+          <%= display_link_if_authorized policy.scap_content.title,
```

Comment:
If you use `display_link_if_authorized` here and the user does not have permission to edit this policy he won't even see the content name (even though he has view granted) so here the `link_to_if_authorized` was correct (same for previous link). The link won't be clickable if user does not have that permission...